instance/static/scss/instance.scss

```diff
@@ -1,3 +1,19 @@
+// Variables
+
+$header-height: 2.8125em;
```

Comment:
I thought this could get picked up easily through the SCSS in `static/` by placing it in the dedicated global variables file, but the global SCSS files aren't actually *global*: they're overridden in the "custom CSS" block with this SCSS file. Adding them to it doesn't seem to help (or maybe I did it incorrectly).

Trying to then instead `@import` them directly in the SCSS file worked beautifully on stage, but then tests were failing because they couldn't find the import, since it wasn't built in the tests.

I finally gave up and just placed it right here. We should really consider either moving all SCSS/JS files to a single place, or ensure that *global* files really are *global*.